src/login/logind.c

```diff
@@ -1171,9 +1171,23 @@ int main(int argc, char *argv[]) {
          * existence of /run/systemd/seats/ to determine whether
          * logind is available, so please always make sure this check
          * stays in. */
-        mkdir_label("/run/systemd/seats", 0755);
-        mkdir_label("/run/systemd/users", 0755);
-        mkdir_label("/run/systemd/sessions", 0755);
+        r = mkdir_label("/run/systemd/seats", 0755);
```

Comment:
`systemd-tmpfiles` [creates](https://github.com/systemd/systemd/blob/d428dd6ac9a56e7b3421fb8ef3aac9937a4a2e62/tmpfiles.d/systemd.conf.m4#L14) `/run/systemd/{seats,users,sessions}` on startup
`systemd-logind` fails:
```
Failed to create /run/systemd/seats directory: File exists
systemd-logind.service: Got notification message from PID 178 (STOPPING=1, STATUS=Shutting down...)
systemd-journald.service: Received EPOLLHUP on stored fd 43 (stored), closing.
Received SIGCHLD from PID 178 (systemd-logind).
Child 178 (systemd-logind) died (code=exited, status=1/FAILURE)
systemd-logind.service: Child 178 belongs to systemd-logind.service
systemd-logind.service: Main process exited, code=exited, status=1/FAILURE
systemd-logind.service: Changed start -> failed
systemd-logind.service: Job systemd-logind.service/start finished, result=failed
```
Seems like we should ignore `EEXIST`
Not sure why don't we call `label_fix`. Hm...